src/containers/ContentLoader.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import ContentLoader from "react-content-loader";
+
+export function ChartLoader({title = 'Loading Data', backgroundColor = "#f3f3f3", 
+	foregroundColor = "#ecebeb", speed = 1.2, width = "100%", height = 150 }, props) {
```

Comment:
Can you please write it on different lines please? I guess that three parameters per line will look good